tests/rados/test_bluestore_configs.py

```diff
@@ -109,3 +111,108 @@ def create_pool_write_iops(param, pool_type):
+            assert int(out.strip("\n")) == 1073741824
+
+            out = mon_obj.get_config(section="osd", param="bluestore_cache_size_ssd")
+            log.info(
+                f"bluestore_cache_size_ssd default value - {out} | Expected 3221225472"
+            )
+            assert int(out.strip("\n")) == 3221225472
+
+            def modify_cache_size(factor):
+                cache_value = int(1073741824 * factor)
+                cache_cfg = {
+                    "section": "osd",
+                    "name": "bluestore_cache_size_hdd",
+                    "value": cache_value,
+                }
+                assert mon_obj.set_config(**cache_cfg)
+                out = mon_obj.get_config(
+                    section="osd", param="bluestore_cache_size_hdd"
+                )
+                log.info(
+                    f"bluestore_cache_size_hdd modified value - {out} | Expected {cache_value}"
+                )
+                assert int(out.strip("\n")) == cache_value
+
+                cache_value = int(3221225472 * factor)
+                cache_cfg = {
+                    "section": "osd",
+                    "name": "bluestore_cache_size_ssd",
+                    "value": cache_value,
+                }
+                assert mon_obj.set_config(**cache_cfg)
+                out = mon_obj.get_config(
+                    section="osd", param="bluestore_cache_size_ssd"
+                )
+                log.info(
+                    f"bluestore_cache_size_ssd modified value - {out} | Expected {cache_value}"
+                )
+                assert int(out.strip("\n")) == cache_value
+
+            # modify ssd and hdd cache (increase)
+            modify_cache_size(factor=1.5)
+
+            # restart osd services
+            restart_osd_service()
+
+            # perform iops
+            create_pool_write_iops(param="cache_inc", pool_type="replicated")
+            create_pool_write_iops(param="cache_inc", pool_type="ec")
+
+            # modify ssd and hdd cache (decrease)
+            modify_cache_size(factor=0.7)
+
+            # restart osd services
+            restart_osd_service()
+
+            # perform iops
+            create_pool_write_iops(param="cache_dec", pool_type="replicated")
+            create_pool_write_iops(param="cache_dec", pool_type="ec")
+
+        except Exception as E:
+            log.error(f"Verification failed with exception: {E.__doc__}")
+            log.error(E)
+            log.exception(E)
+            return 1
+        finally:
+            # reset modified cache configs
+            mon_obj.remove_config(
+                **{"section": "osd", "name": "bluestore_cache_size_hdd"}
+            )
+            mon_obj.remove_config(
+                **{"section": "osd", "name": "bluestore_cache_size_ssd"}
+            )
+
+            # restart osd services
+            restart_osd_service()
+            wait_for_clean_pg_sets(rados_obj, timeout=300, _sleep=10)
+
```

Comment:
Please remove the pools.

Reply:
Pools are already removed post IOPS and verification as part of `def create_pool_write_iops(param, pool_type)` Line 49